Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -701,20 +705,29 @@ window.__firefox__.execute(function($) {
       scheduleQueuePump(false, false)
     }
   }
-
-  CC.tryScheduleQueuePump = CC.tryScheduleQueuePump || tryScheduleQueuePump
-  tryScheduleQueuePump()
   
   // Wait until document head is ready. We just need to wait a moment
   window.setTimeout(function () {
```

Comment:
Is this documentation correct? I don't see where exactly it is waiting for `head` to be ready?
This script is injected inside of `SiteStateListenerScriptHandler`'s message handler, via `evaluateJavascript` which executes when `document` is already loaded.
`SiteStateListenerScriptHandler` is injected at `documentStart`, so after `head` has already loaded and when the document has loaded.

Why do we need this timeout?

Reply:
ahh yes an old comment from the old way it was injected I believe. Not even sure if this timeout is all that necessary anymore.